packages/design-system-dashboard-cli/test/search-files.spec.js

```diff
@@ -18,7 +18,10 @@ const json = {
   '../.env.json': JSON.stringify(env),
   'mock-vw/jsx-file.jsx': jsxFile,
   'mock-vw/js-file.js': 'some JS',
-  // 'mock-vw/js-file.unit.js': 'Testing code',
+  'mock-vw/app/js-file.unit.js': 'Testing file',
+  'mock-vw/app/js-file.spec.js': 'Testing file',
+  'mock-vw/app/js-file.unit.spec.js': 'Testing file',
+  'mock-vw/app/tests/js-file.js': 'Testing file',
```

Comment:
Should I add tests to show that these are explicitly not included?

Reply:
No, I feel like your test covers this.